Representable/representable
# Add organization, profile link to navbar

Key Changes:
- Redesigns navbar as per spec below:
<img width="774" alt="Screen Shot 2020-08-26 at 2 19 50 AM" src="https://user-images.githubusercontent.com/8892509/91285912-a0cf0100-e742-11ea-97da-75eb02305f57.png">
- Adds logic to user model to get orgs (get_organizations) they're an admin of, uses this to display orgs on navbar
- Still Todo: account settings revamp
- Also Todo: add translation for new navbar links: profile, account settings

To test:
- Make sure website works with multiple orgs, no orgs (but logged in), and not logged in

Screen of navbar:
<img width="767" alt="Screen Shot 2020-08-26 at 2 17 51 AM" src="https://user-images.githubusercontent.com/8892509/91286720-a6791680-e743-11ea-9e5f-fae71a9ada1d.png">
